File: app/src/main/java/jp/naist/rocatmonitor/androcatmonitorxp/message/ExeTimeInfo.java
package jp.naist.rocatmonitor.androcatmonitorxp.message;

@org.msgpack.annotation.Message
public class ExeTimeInfo
{

  @org.msgpack.annotation.Index(0)
  public long MethodID = 0;

  @org.msgpack.annotation.Index(1)
  public long ThreadID = 0;

  @org.msgpack.annotation.Index(2)
  public long ExeTime = 0;

  public ExeTimeInfo()
  {
  }

  public ExeTimeInfo(long methodID, long threadID, long exeTime)
  {
    MethodID = methodID;
    ThreadID = threadID;
    ExeTime = exeTime;
  }

}

File: project/RocatMonitorVisualizer/Assets/Scripts/Message/ExeTimeInfo.cs
using MsgPack.Serialization;

public class ExeTimeInfo
{

  [MessagePackMember(0)]
  public int MethodID = 0;

  [MessagePackMember(1)]
  public long ThreadID = 0;

  [MessagePackMember(2)]
  public double ExeTime = 0;

}